### ISSUE TITLE
The simulator fails to load Harvester v1.4-head generated SBs

### ISSUE THREAD

**Issue by bk201** [rancher/support-bundle-kit#123]
I'm seeing the error while loading a v1.4-head SB:

```
I1008 17:25:46.906142   10415 handler.go:275] Adding GroupVersion kubevirt.io v1 to ResourceManager
I1008 17:25:46.906245   10415 handler.go:275] Adding GroupVersion kubevirt.io v1alpha3 to ResourceManager
I1008 17:25:47.065752   10415 handler.go:275] Adding GroupVersion snapshot.kubevirt.io v1alpha1 to ResourceManager
I1008 17:25:47.133415   10415 handler.go:275] Adding GroupVersion snapshot.kubevirt.io v1alpha1 to ResourceManager
I1008 17:25:47.342131   10415 handler.go:275] Adding GroupVersion harvesterhci.io v1beta1 to ResourceManager
I1008 17:25:47.620626   10415 handler.go:275] Adding GroupVersion harvesterhci.io v1beta1 to ResourceManager
I1008 17:25:47.731032   10415 handler.go:275] Adding GroupVersion network.harvesterhci.io v1beta1 to ResourceManager
I1008 17:25:47.950796   10415 handler.go:275] Adding GroupVersion network.harvesterhci.io v1beta1 to ResourceManager
I1008 17:25:48.134692   10415 handler.go:275] Adding GroupVersion longhorn.io v1beta2 to ResourceManager
I1008 17:25:48.362937   10415 handler.go:275] Adding GroupVersion longhorn.io v1beta1 to ResourceManager
I1008 17:25:48.363022   10415 handler.go:275] Adding GroupVersion longhorn.io v1beta2 to ResourceManager
I1008 17:25:48.551537   10415 handler.go:275] Adding GroupVersion snapshot.storage.k8s.io v1 to ResourceManager
[Step 1/4: Cluster CRDs] 161/161
Time to load all objects: 30.022340727s seconds

I1008 17:25:48.752744   10415 handler.go:275] Adding GroupVersion snapshot.storage.k8s.io v1 to ResourceManager
I1008 17:25:48.951586   10415 handler.go:275] Adding GroupVersion snapshot.storage.k8s.io v1 to ResourceManager
FATA[0040] Error loading cluster scoped objects error looking up GVR no matches for kind "FlowSchema" in version "flowcontrol.apiserver.k8s.io/v1" for object &{map[apiVersion:flowcontrol.apiserver.k8s.io/v1 kind:FlowSchema metadata:map[annotations:map[apf.kubernetes.io/autoupdate-spec:true sim.harvesterhci.io/creationTimestamp:2024-10-08T07:42:53Z] creationTimestamp:2024-10-08T07:42:53Z generation:1 managedFields:[map[apiVersion:flowcontrol.apiserver.k8s.io/v1 fieldsType:FieldsV1 fieldsV1:map[f:status:map[f:conditions:map[k:{"type":"Dangling"}:map[.:map[] f:lastTransitionTime:map[] f:message:map[] f:reason:map[] f:status:map[] f:type:map[]]]]] manager:api-priority-and-fairness-config-consumer-v1 operation:Apply subresource:status time:2024-10-08T07:42:53Z] map[apiVersion:flowcontrol.apiserver.k8s.io/v1 fieldsType:FieldsV1 fieldsV1:map[f:metadata:map[f:annotations:map[.:map[] f:apf.kubernetes.io/autoupdate-spec:map[]]] f:spec:map[f:distinguisherMethod:map[.:map[] f:type:map[]] f:matchingPrecedence:map[] f:priorityLevelConfiguration:map[f:name:map[]] f:rules:map[]]] manager:api-priority-and-fairness-config-producer-v1 operation:Update time:2024-10-08T07:42:53Z]] name:catch-all uid:0441f1d5-4f5e-4d16-88a0-bfae2bf9300e] spec:map[distinguisherMethod:map[type:ByUser] matchingPrecedence:10000 priorityLevelConfiguration:map[name:catch-all] rules:[map[nonResourceRules:[map[nonResourceURLs:[*] verbs:[*]]] resourceRules:[map[apiGroups:[*] clusterScope:true namespaces:[*] resources:[*] verbs:[*]]] subjects:[map[group:map[name:system:unauthenticated] kind:Group] map[group:map[name:system:authenticated] kind:Group]]]]] status:map[conditions:[map[lastTransitionTime:2024-10-08T07:42:53Z message:This FlowSchema references the PriorityLevelConfiguration object named "catch-all" and it exists reason:Found status:False type:Dangling]]]]}
```

**Comment by bk201** [comment]
[supportbundle_dba96edf-1644-4a02-92be-b5a2dddef8ae_2024-10-08T09-22-36Z.zip](https://github.com/user-attachments/files/17291177/supportbundle_dba96edf-1644-4a02-92be-b5a2dddef8ae_2024-10-08T09-22-36Z.zip)

**Comment by bk201** [comment]
Fix by #125.
